smarts/core/sensors.py

```diff
@@ -111,6 +114,12 @@ class DrivableAreaGridMap(NamedTuple):
     data: np.ndarray
 
 
+@dataclass
+class ViaData:
+    near_via_lanes: List[Tuple[Via, int]]
```

Comment:
Maybe adding a comment about `near_via_lanes` and `on_via_lanes` to help understand the code?

Reply:
I added an example and description.